acquire/acquire.py

```diff
@@ -1453,6 +1453,16 @@ class Home(Module):
         ("dir", ".config", from_user_home),
         ("glob", "*/.*[akz]sh*", from_user_home),
         ("glob", "*/.config", from_user_home),
+        ("glob", ".*history", from_user_home),
+        ("glob", "*/.*history", from_user_home),
+        ("glob", ".*rc", from_user_home),
+        ("glob", ".*_logout", from_user_home),
```

Comment:
Maybe add a small comment above `.*[akz]sh*` that this should collect most shell configuration files, and that with these you try to collect some that aren't collected with that crazy glob?